demo/app/ios/Runner/OpenID4CI.swift

```diff
@@ -31,14 +31,53 @@ public class OpenID4CI {
         self.initiatedInteraction = Openid4ciNewInteraction(args, opts, nil)!
     }
     
+    func checkFlow() throws -> String {
+        let issuerCapabilities = initiatedInteraction.issuerCapabilities()
+        if ((issuerCapabilities!.authorizationCodeGrantTypeSupported())){
+            return "auth-code-flow"
+        }
+        if ((issuerCapabilities!.preAuthorizedCodeGrantTypeSupported())){
+           return "preauth-code-flow"
+        }
+        return ""
+    }
+    
+    func getAuthorizationLink() throws -> String {
+        let issuerCapabilities = initiatedInteraction.issuerCapabilities()
+        if !(issuerCapabilities?.authorizationCodeGrantTypeSupported())! {
+            // This code example isn't applicable. See the authorization code flow example instead.
+            return "Not implemented"
```

Comment:
Should throw an error

Reply:
error handling I am doing separately in this pr #426 as I have to catch properly in the flutter layer and show error too